docs/rules/idiomatic/use-object-keys.md

```diff
@@ -0,0 +1,53 @@
+# use-object-keys
+
+**Summary**: Prefer to use `object.keys`
+
+**Category**: Idiomatic
+
+**Automatically fixable**: Yes/No
```

Comment:
```suggestion
**Automatically fixable**: No
```

Reply:
Thanks! I removed that entirely as we don't normally show this when fixing is unsupported. Adding a fix for this seems simple though, so might be that I look into that next.